src/components/pages/about.vue

```diff
@@ -34,218 +31,36 @@
 </template>
 
 <script>
-import MainContentDropdown from '../molecules/about/main-content-dropdown.vue';
-import MainContentHistory from '../molecules/about/main-content-history.vue';
+import MainContentDropdown from "../molecules/about/main-content-dropdown.vue";
+import MainContentHistory from "../molecules/about/main-content-history.vue";
+import { aboutContent as staticDescriptions } from "../static/data/index.js";
```

Comment:
```suggestion
import { aboutContent as staticDescriptions } from "../static/data";
```
위와 같이 index.js 를 숨기는 것은 어떠할지 제안드려봅니다.